src/test/java/com/bandwidth/sqs/consumer/SqsConsumerTest.java

```diff
@@ -99,6 +99,7 @@ public SqsConsumerTest() {
                 .withBackoffStrategy(backoffStrategyMock)
                 .withExpirationStrategy(expirationStrategyMock)
                 .withAutoExpire(true)
+                .withShutdownTimeout(SqsConsumer.DEFAULT_SHUTDOWN_TIMEOUT)
```

Comment:
Why is this needed? Doesn't it use the default... by default?

Reply:
Coverage